third_party/move/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -130,9 +135,10 @@ fn annotated_module_from_unit(unit: &AnnotatedCompiledUnit) -> Option<&Annotated
     }
 }
 
-impl PreCompiledModules for FullyCompiledProgram {
+impl PreCompiledModules for (FullyCompiledProgram, Vec<PackagePaths>) {
```

Comment:
I did not even know we can do something like this.

Reply:
I only tried it on a whim, and was surprised to find it compiles.